src/igvfd/upgrade/file_set.py

```diff
@@ -184,3 +184,15 @@ def measurement_set_11_12(value, system):
     if 'protocol' in value:
         value['protocols'] = [value['protocol']]
         del value['protocol']
+
+
+@upgrade_step('prediction_set', '3', '4')
```

Comment:
since the behavior in construct library changed, you need to upgrade the schema version as well. and reflect in change log as well

Reply:
and do still agree with you, there is no need in notes update here

Reply:
Ok but this will be an empty upgrade as nothing has been modified within the schema.